### ISSUE TITLE
procstat input isn't tagging cgroup or systemd_unit

### ISSUE THREAD

**Issue by daytonjones** [influxdata/telegraf#6379]
### Relevant telegraf.conf:

```[[inputs.procstat]]
  user = "rails"
  systemd_unit = "apache2.service"
  cgroup = "systemd/system.slice/apache2.service"

```

### System info:
Telegraf 1.12.0 
Ubuntu 16.04 LTS

### Steps to reproduce:

1. Add config
2. start telegraf

### Expected behavior:
Expected tags for systemd_unit and cgroup
### Actual behavior:
Those tags don't exist:
'''procstat_lookup,host=myhostname,pid_finder=pgrep,result=success,user=rails'''
'''procstat,host=myhostname,process_name=ruby,user=rails'''

### Additional info:
'''
> show tag keys from "procstat_lookup"
name: procstat_lookup
tagKey
------
host
pid_finder
pidfile
result
user

> show tag keys from "procstat"
name: procstat
tagKey
------
host
pidfile
process_name
user
'''

**Comment by glinton** [comment]
I don't believe you can have `user`, `systemd_unit`, and `cgroup` defined, as telegraf will [just use one](https://github.com/influxdata/telegraf/blob/release-1.12/plugins/inputs/procstat/procstat.go#L371). Hence why there is a `user` tag, but not anything else.

Perhaps this is a docs issue?
